src/Tests/Directory.Build.targets

```diff
@@ -0,0 +1,22 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  
+  <ItemGroup Condition="'$(IsTestProject)' == 'true' AND '$(OutputType)' == 'Exe'">
+    <Compile Include="$(MSBuildThisFileDirectory)Common\Program.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsTestProject)' == 'true' AND '$(OutputType)' == 'Exe'">
+
+    <!-- Note: These items are showing up in solution explorer even though Visible is set to false -->
```

Comment:
Is there a project system bug tracking this? cc @davkean

Reply:
I don't know what the bug is - the items show up? Or just the folder?

Reply:
@davkean Turns out it's just the folders that show up:

![image](https://user-images.githubusercontent.com/145043/39949273-b6f68faa-552e-11e8-8fdb-a0c8f47df456.png)

Reply:
Yeah we have a bug on this.